apps/antalmanac/src/components/RightPane/SectionTable/SectionTable.tsx

```diff
@@ -10,109 +12,122 @@ import {
-        },
-        '&:nth-child(5)': {
-            width: '12%',
-        },
-        '&:nth-child(6)': {
-            width: '10%',
-        },
-        '&:nth-child(7)': {
-            width: '10%',
-        },
-        '&:nth-child(8)': {
-            width: '8%',
-        },
-        '&:nth-child(9)': {
-            width: '8%',
-        },
+    gpa: {
+        label: 'GPA',
+        width: '6%',
+    },
+    dayAndTime: {
+        label: 'Times',
+        width: '15%',
+    },
+    location: {
+        label: 'Places',
+        width: '8%',
+    },
+    sectionEnrollment: {
+        label: 'Enrollment',
+        width: '9%',
+    },
+    restrictions: {
+        label: 'Restr',
+        width: '8%',
+    },
+    status: {
+        label: 'Status',
+        width: '8%',
     },
-    container: {},
-    titleRow: {},
-    clearSchedule: {},
-    scheduleNoteContainer: {},
-};
-
-const tableHeaderColumns: Record<SectionTableColumn, string> = {
-    sectionCode: 'Code',
-    sectionDetails: 'Type',
-    instructors: 'Instructors',
-    dayAndTime: 'Times',
-    location: 'Places',
-    sectionEnrollment: 'Enrollment',
-    restrictions: 'Restr',
-    status: 'Status',
 };
+const tableHeaderColumnEntries = Object.entries(tableHeaderColumns);
 
-const SectionTable = (props: SectionTableProps) => {
-    const { classes, courseDetails, term, allowHighlight, scheduleNames, analyticsCategory } = props;
-    const courseId = courseDetails.deptCode.replaceAll(' ', '') + courseDetails.courseNumber;
-    const encodedDept = encodeURIComponent(courseDetails.deptCode);
-    const isMobileScreen = useMediaQuery(`(max-width: ${MOBILE_BREAKPOINT}`);
+interface EnrollmentColumnHeaderProps {
+    label: string;
+}
 
-    const [activeColumns, setActiveColumns] = useState(RightPaneStore.getActiveColumns());
+function EnrollmentColumnHeader(props: EnrollmentColumnHeaderProps) {
+    const isMobileScreen = useMediaQuery(`(max-width: ${MOBILE_BREAKPOINT})`);
 
-    const handleColumnChange = useCallback(
-        (newActiveColumns: SectionTableColumn[]) => {
-            setActiveColumns(newActiveColumns);
-        },
-        [setActiveColumns]
+    return (
+        <Box display="flex">
+            {props.label}
+            {!isMobileScreen && (
+                <Tooltip
+                    title={
+                        <Typography>
+                            Enrolled/Capacity
+                            <br />
+                            Waitlist
+                            <br />
+                            New-Only Reserved
+                        </Typography>
+                    }
+                >
+                    <Help fontSize="small" />
+                </Tooltip>
+            )}
+        </Box>
     );
+}
+
+function SectionTable(props: SectionTableProps) {
+    const { courseDetails, term, allowHighlight, scheduleNames, analyticsCategory } = props;
+
+    const { selectedColumns, getActiveColumns } = useColumnStore();
```

Comment:
You can slice the store. A minor visual improvement is that the left side looks more like a typical useState call.

```suggestion
    const [ selectedColumns, getActiveColumns ] = useColumnStore(store => [store.selectedColumns, store.getActiveColumns]);
```